codegen/sdk-codegen/build.gradle.kts

```diff
@@ -206,11 +206,15 @@ task("stageSdks") {
     group = "codegen"
     description = "relocate generated SDK(s) from build directory to Sources and Tests directories"
     doLast {
+        val withManifests = project.properties["withManifests"] != null
```

Comment:
If you run the stageSdks Gradle task with the option `-PwithManifests`, this will be true and `Package.swift` will be copied over for ever service client